[CI] IndicesRequestIT testIndicesStats failing

**Build Scans:**
- [elasticsearch-intake #30833 / part1](https://gradle-enterprise.elastic.co/s/2st6kwgzxnh2i)
- [elasticsearch-intake #30748 / part1](https://gradle-enterprise.elastic.co/s/timraeh47nvg2)

**Reproduction Line:**
```
./gradlew ":server:internalClusterTest" --tests "org.elasticsearch.action.IndicesRequestIT.testIndicesStats" -Dtests.seed=5C54431A6A574E4F -Dtests.locale=hy-Armn-AM -Dtests.timezone=Canada/Yukon -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.action.IndicesRequestIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testIndicesStats'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: org.elasticsearch.action.support.broadcast.node.TransportBroadcastByNodeAction$NodeRequest
Expected: ["test0", "test0-alias"]
     but: was []
```

**Issue Reasons:**
- [main] 2 failures in test testIndicesStats (0.6% fail rate in 348 executions)
- [main] 2 failures in step part1 (3.8% fail rate in 52 executions)
- [main] 2 failures in pipeline elasticsearch-intake (3.8% fail rate in 52 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -429,9 +429,6 @@ tests:
 - class: org.elasticsearch.xpack.ilm.CCRIndexLifecycleIT
   method: testTsdbLeaderIndexRolloverAndSyncAfterWaitUntilEndTime {targetCluster=FOLLOWER}
   issue: https://github.com/elastic/elasticsearch/issues/137565
-- class: org.elasticsearch.action.IndicesRequestIT
-  method: testIndicesStats
-  issue: https://github.com/elastic/elasticsearch/issues/137594
 - class: org.elasticsearch.xpack.esql.vector.VectorSimilarityFunctionsIT
   method: testTopNSimilarityBetweenConstantVectorAndField {functionName=v_l2_norm
     similarityFunction=org.elasticsearch.xpack.esql.expression.function.vector.L2Norm$1@5b068087 elementType=byte}

--- a/server/src/internalClusterTest/java/org/elasticsearch/action/IndicesRequestIT.java
+++ b/server/src/internalClusterTest/java/org/elasticsearch/action/IndicesRequestIT.java
@@ -131,6 +131,7 @@ public class IndicesRequestIT extends ESIntegTestCase {
             // InternalClusterInfoService sends IndicesStatsRequest periodically which messes with this test
             // this setting disables it...
             .put("cluster.routing.allocation.disk.threshold_enabled", false)
+            .put("cluster.routing.allocation.write_load_decider.enabled", "disabled")
             .build();
     }
 

